src/tests/class_tests/openms/source/AASequence_test.cpp

```diff
@@ -232,8 +232,17 @@ START_SECTION((double getAverageWeight(Residue::ResidueType type = Residue::Full
 END_SECTION
 
 START_SECTION((double getMonoWeight(Residue::ResidueType type = Residue::Full, Int charge=0) const))
-  TOLERANCE_ABSOLUTE(1e-6)
-  TOLERANCE_RELATIVE(1.0 + 1e-6)
+  TOLERANCE_ABSOLUTE(1e-5)
+  TOLERANCE_RELATIVE(1.0 + 1e-5)
```

Comment:
is this an new issue of numerical instability or are the numbers just different from what we had before?
In either case, we must ensure that the numbers are accurate since modern high res instruments can easily tell the difference of one electron mass (~1/2000 amu)